### ISSUE TITLE
network server with link grammar jar??

### ISSUE THREAD

**Issue by dijs** [dijs/link-grammar#8]

**Comment by linas** [comment]
There already is a "link-grammar server jar" here:  https://github.com/opencog/link-grammar/blob/master/bindings/java/link-grammar-server.sh -- it spews out json.

**Comment by linas** [comment]
Note also: We use link-grammar as the first stage to a piepline, the second stage is relex; it computes dependency-grammar relations. It also has several network servers; I forget; one might json, maybe, I don't recall. relex is here:  https://github.com/opencog/relex  documentation is here: http://wiki.opencog.org/w/RelEx

**Comment by dijs** [comment]
The only downside to use is the Java dependency... I would much rather get
the native ffi linking working better

**Comment by linas** [comment]
Yes, sure, that makes sense, Thought I'd point it out.  (we're kind-of trying to get rid of java too).

Currently the ffi for link-grammar is generated by/managed by SWIG  The FFI definition is here: https://github.com/opencog/link-grammar/blob/master/bindings/swig/link_grammar.i  If you know of a better ffi tool, or prefer something different, ... I'll happily include that as a part of the LG distribution.
